src/fragments/lib/logging/common/change_log_levels/20_category_log_level.mdx

```diff
@@ -0,0 +1,22 @@
+Add a `categoryLogLevel` section and specify each category and its log level.
+
+```json
+{
+    "awsCloudWatchLoggingPlugin": {
+        "enable": true,
+        "logGroupName": "<log-group-name>",
+        "region": "<region>",
+        "localStoreMaxSizeInMB": 1 MB,
```

Comment:
🤔 shouldn't this just be `1`, without the `MB`?
```suggestion
        "localStoreMaxSizeInMB": 1,
```

Reply:
Removed `MB`